### ISSUE TITLE
Missing toggle for the accessibility tooltip

### ISSUE THREAD

**Issue by rdw-software** [desktop/desktop#20898]
### The feature request

The tooltip disrupts my workflow as it constantly pops up while hovering commits (list items). Popping in visual elements is quite distracting and, perhaps ironically, not very accessible for that reason alone. This problem is made worse by the fact that multiple tooltips show up (see #20896), but even so I'd prefer not having a tooltip at all.

### Proposed solution

This is the location where I would expect to find a setting to disable the feature:

<img width="566" height="378" alt="Image" src="https://github.com/user-attachments/assets/6ea7a2d0-a577-43e1-96b7-ed0487038bbf" />

### Additional context

I'd welcome it if you stopped introducing UI changes without an option to turn them off. It is not good practice and likely to aggravate developers, who as power-users can be very particular about how their environment behaves.
